### DISCUSSION TITLE
2.5.4 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#1715] [讨论]
· 正版登录调整
　　彻底移除 Mojang 登录
　　在未登录的页面添加了 [购买正版] 与 [老账号迁移] 选项
　　在已登录的页面添加了 [修改玩家名] 选项
　　重新制作选项界面，将皮肤更改等功能整理到了一起
　　如果从未登录过正版账号，会偶尔建议购买正版……
　　更换正版与离线验证的图标（#1637，@Chiloven945）
· 细节调整与优化
　　修改秋仪金的解锁提示（#1639，@404nfwpz）
　　调整微软账号后登录失败的弹窗内容（#1675，@sleep80）
　　将 RC 版本的描述修改为“发布候选”（#1680，@iPodtouch6557）
· 修复了以下 14 个 Bug
　　页面滚动在指向文本框或下拉框时表现异常（#1563，@404nfwpz）
　　同时安装 OptiFine 与 Forge 有极小概率失败（#1627，@Bandysol）
　　在启动完成的同时取消启动不会结束游戏进程（#1631，@Bandysol）
　　错误地允许使用 Java 20 启动 1.19+ Forge（#1635，@Dave12138）
　　特殊情况下可能会使更新启动器失败（#1641，@YUANXINGlingyun）
　　部分离线皮肤会被错误地缩放（#1656，@gaojin20090517）
　　子 Mixin 导致的崩溃分析可能不显示 Mod 名称（#1666，@tcybsw）
　　点击特定像素可以进入错误的 Mod 管理页面（#1668，@youzi-2333）
　　自定义启动命令的起始文件夹有误（#1674，@QianQiuawa）
　　部分整合包被错误地指向 Mod（#1676，@Chiloven945）
　　无法加载 1.19 ~ 1.19.2 自定义离线皮肤（#1707，@ExpertPbSeal）
　　使用双层转单层的离线皮肤时会导致皮肤模糊
　　图标按钮在鼠标移开后的颜色有误

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.5.4'>2.5.4</a>.</em>

**Comment by Chormore-EDGE** [comment]
mojang：6

**Comment by iPodtouch6557** [comment]
好耶~更新啦

**Comment by Chormore-EDGE** [comment]
~为什么感觉 #1637 的意思是改成Windows11徽标~

**Reply by LTCatt** [reply]
这样好看点.jpg

**Reply by Chiloven945** [reply]
~准确地说是微软徽标（~

**Reply by Dave-12138** [reply]
~~为什么github给我发的邮件里有删除线的字体是蓝绿色的~~

![](https://s2.loli.net/2023/04/27/p284cEQLPqwaZus.png)

**Reply by iPodtouch6557** [reply]
> ~为什么github给我发的邮件里有删除线的字体是蓝绿色的~
> 
> ![](https://camo.githubusercontent.com/312ce450deb1026b881fbd1207d55fa4d6913f6c40db2015f4d854dcad32bd8b/68747470733a2f2f73322e6c6f6c692e6e65742f323032332f30342f32372f703238346345514c507177615a75732e706e67)

一直都是
甚至我收到的划线都是蓝绿色的

**Reply by Chiloven945** [reply]
我这边是红色（

**Reply by Chormore-EDGE** [reply]
![image](https://user-images.githubusercontent.com/114737601/235333228-9bcfea0c-51d2-4b27-8a46-38473e0c80d6.png)
杀软：9

**Reply by Chiloven945** [reply]
> ![image](https://user-images.githubusercontent.com/114737601/235333228-9bcfea0c-51d2-4b27-8a46-38473e0c80d6.png) 杀软：9

的确有一种感觉很像defender（~或者说就是defender~
